src/components/CategoryOverview/CategoryOverview.test.js

```diff
@@ -68,8 +68,8 @@ describe('when rendering the "context" category', () => {
     expect(component.find('.categoryWrapper').hasClass('category__context')).toBe(true);
   });
 
-  it('should not render a category icon', () => {
-    expect(component.find('.categoryIcon').length).toBe(0);
+  it('should render a category icon', () => {
+    expect(component.find('.categoryIcon').length).toBe(1);
```

Comment:
This test can actually go since it's no longer conditional on the context category (there's a few other tests in here that are no longer conditional that can be moved/removed)